Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                        buttons: [
+                            backButton,
+                            nextButton,
+                        ],
+                        id: 'site_setup',
+                    },
+                    {
+                        title: 'Log in',
+                        text: 'Let\'s log in! Please go to the following URL <i>"~/Login"</i>, by clicking on the' +
+                            ' <i>"Next"</i> button.',
+                        buttons: [
+                            backButton,
+                            {
+                                action: function () {
+                                    goToRelativePage(Shepherd.activeTour.options.id, 'login_page', null, 'Login');
+                                },
+                                classes: 'shepherd-button-primary',
+                                text: 'Next',
+                            },
+                        ],
+                        id: 'logging_in',
+                    },
+                    {
+                        title: 'Log in page',
+                        text: 'Here you can log in. You will need to input a username or email address and a password.',
+                        buttons: [
+                            {
+                                action: function () {
+                                    goToRelativePage(Shepherd.activeTour.options.id, 'logging_in', 'Login');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'login_page',
+                        when: {
+                            show() {
+                                // If the user typed a wrong URL don't go ahead.
+                                // The best way to check this, is to check for specific element(s). We don't want to
+                                // check the URL, since the case can vary and also, we can't account for the tenant.
+                                if (!$('button[type="Submit"]').length) {
+                                    deleteWalkthroughCookies();
+                                    Shepherd.activeTour.back();
+                                    return;
+                                }
+
+                                // If the user presses enter, the form will be submitted and the steps will be skipped.
+                                preventSubmit();
+                            },
+                        },
+                    },
+                    {
+                        title: 'Username',
+                        attachTo: { element: '#UserName', on: 'bottom' },
+                        text: 'Input your username, for test sites the usually used username is <i>"admin"</i>.',
```

Comment:
```suggestion
                        text: 'Input your username, for this sites the used username is <i>"admin"</i>.',
```

Reply:
Create this user from the recipe. You can use the following command: https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Users/Commands/UserCommands.cs#L38

Reply:
Or it would be even better to use something else for username and password so if we create a tenant with autosetup we can still use our usual stuff.